pages/register.tsx

```diff
@@ -253,65 +282,178 @@ export default function Register() {
-                <DisplayQuestion key={idx} obj={obj} values={values} onChange={handleChange} />
-              ))}
-
-              <div className="text-2xl py-1 border-b-2 border-black mr-auto mt-8">Event Info</div>
-              {eventInfoQuestions.map((obj, idx) => (
-                <DisplayQuestion key={idx} obj={obj} values={values} onChange={handleChange} />
-              ))}
-
-              <div className="text-2xl py-1 border-b-2 border-black mr-auto mt-8">Sponsor Info</div>
-              {sponsorInfoQuestions.map((obj, idx) => (
-                <DisplayQuestion key={idx} obj={obj} values={values} onChange={handleChange} />
-              ))}
-
-              {/* Resume Upload */}
-              <label className="mt-4">
-                Upload your resume:
-                <br />
-                <input
-                  onChange={(e) => handleResumeFileChange(e)}
-                  name="resume"
-                  type="file"
-                  formEncType="multipart/form-data"
-                  accept=".pdf, .doc, .docx, image/png, image/jpeg, .txt, .tex, .rtf"
-                />
-                <br />
-              </label>
-
-              {/* Submit */}
-              <div className="my-8">
-                <button
-                  type="submit"
-                  className="mr-auto cursor-pointer px-4 py-2 rounded-md bg-blue-200 hover:bg-blue-300"
-                  onClick={() => setFormValid(!(!isValid || !dirty))}
-                >
-                  Submit
-                </button>
-                {!isValid && !formValid && (
-                  <div className="text-red-600">Error: The form has invalid fields</div>
-                )}
-              </div>
+              {/* General Questions */}
+              {registrationSection == 0 && (
+                <section className="bg-white lg:w-3/5 md:w-3/4 w-full min-h-[35rem] mx-auto rounded-2xl md:py-10 py-6 px-8 mb-8 text-[#4C4950]">
+                  Please fill out the following fields. The application should take approximately 5
+                  minutes.
+                  <h2 className="sm:text-2xl text-xl font-semibold mt-5 sm:mb-3 mb-1">General</h2>
+                  <div className="flex flex-col">
+                    {generalQuestions.map((obj, idx) => (
+                      <DisplayQuestion
+                        key={idx}
+                        obj={obj}
+                        values={values}
+                        onChange={handleChange}
+                      />
+                    ))}
+                  </div>
+                </section>
+              )}
+
+              {/* School Questions */}
+              {registrationSection == 1 && (
+                <section className="bg-white lg:w-3/5 md:w-3/4 w-full min-h-[35rem] mx-auto rounded-2xl md:py-10 py-6 px-8 mb-8 text-[#4C4950]">
+                  Please fill out the following fields. The application should take approximately 5
+                  minutes.
```

Comment:
lmao is this supposed to be an actual estimate? Probably should mark that as TODO/FIXME so that the feature gets implemented at some point.
If there isn't a plan to make this an actual estimate, it feels weird to include such a specific estimation.

Reply:
idk this was just here since the very beginning on figma lol

Reply:
Okay we can just leave it then.